docs/index.rst

```diff
@@ -29,6 +29,18 @@ allow for that.
 Overall, skorch aims at being as flexible as PyTorch while
 having a clean interface as sklearn.
 
+If you use skorch, please use this BibTeX entry:
+
+.. code:: bibtex
+
+   @manual{skorch,
+     author       = {Marian Tietz and Thomas J. Fan and Daniel Nouri and Benjamin Bossan and {skorch Developers}},
```

Comment:
I would put Nouri and Bossan ahead of me on this list.

```suggestion
     author       = {Marian Tietz and Daniel Nouri and Benjamin Bossan and Thomas J. Fan and {skorch Developers}},
```

Reply:
Thomas, you are too modest.